apps/new/widget/page/projects/Editor.jsx

```diff
@@ -51,12 +51,16 @@ const isNearAddress = (address) => {
 
 const tabs = [
   { id: "overview", label: "Overview", disabled: true, checked: true },
-  { id: "activity", label: "Activity", checked: true },
+  // { id: "activity", label: "Activity", checked: true },
   { id: "tasks", label: "Tasks", checked: true },
   // Uncomment after the support is added
   // { id: "discussion", label: "Discussion", checked: false },
   // { id: "code", label: "Code", checked: false },
   // { id: "roadmap", label: "Roadmap", checked: false },
+  // Feed tabs
+  { id: "allFeed", label: "All Feed", checked: true },
```

Comment:
may be we can rename it to `activity`?
what do you think?

Reply:
```{ id: "activity", label: "Activity", checked: true },```